### PR TITLE
Fix: multiple broken links

### PR DESCRIPTION
**Pull request by lwasser**

### CODE REVIEW

File: tutorials/add-readme.md [hunk 1]
```diff
@@ -223,7 +222,7 @@ above is a set of recommendations as you are just getting started. You may find
 the need for other elements to be added to this file as you further develop your
 package and as a community begins to use your package.
 
-In the [next lesson](add-license-coc), you will add a LICENSE file to
+In the [next lesson](add-license-coc.md), you will add a LICENSE file to
```
Comment (lwasser) [inline tutorials/add-readme.md:225]: This link will be published friday of this week.

**Review by willingc — APPROVED** [review]
LGTM. Thanks!

File: tutorials/add-readme.md (hunk 1 above)
Comment (willingc) [reply tutorials/add-readme.md:225]: Failing htmlproofer in CI now but should pass once the link is up.

**Comment by lwasser** [comment]
ok merging! thanks for the review @willingc !!

**Pull request merged**